src/components/HeaderAuth.jsx

```diff
@@ -50,10 +45,9 @@ HeaderAuth.defaultProps = {
   initialised: false,
 };
 
-function mapStateToProps(state) {  // Listens for changes in the Redux Store
-  return {
-    user: state.login.user,
-    initialised: state.login.initialised,
-  };
-}
+const mapStateToProps = (state) => ({
```

Comment:
I think it's called Module Pattern https://developer.mozilla.org/en/docs/Web/JavaScript/Closures#Emulating_private_methods_with_closures